src/plugins/es_ui_shared/static/forms/components/fields/combobox_field.tsx

```diff
@@ -68,6 +68,12 @@ export const ComboBoxField = ({ field, euiFieldProps = {}, idAria, ...rest }: Pr
     }
   };
 
```

Comment:
I added this change to the PR because I noticed that `ArrayItems` validations in `ComboBox` are only executed on every item but `field` validations are not taken into account. That doesn't allow us to make a`ComboBox` a required field. So I am executing validations of type `field` on `onBlur`. After this change we can apply a validation like the following to the `ComboBox`.

```javascript
{
    validator: emptyField('The field must not be empty'),
    isBlocking: true,
    type: 'field'
}
 ```

Reply:
I am not sure I follow the issue you found. We have many forms where we don't allow empty array for the ComboBox and didn't need this  extra `onBlur`. You can see one example [here](https://github.com/elastic/kibana/blob/master/x-pack/plugins/index_management/public/application/components/mappings_editor/constants/parameters_definition.tsx#L896).

The validation `type: "field"` is executed whenever the field value changes (which here is on L58 & L62). Can you explain why we need the `onBlur`?